### DISCUSSION TITLE
My GitHub Actions workflows are not starting

### DISCUSSION THREAD

**Discussion by giscus** [rajbos/rajbos.github.io#14] [Comments]
# My GitHub Actions workflows are not starting

My GitHub Actions workflows are not starting

https://devopsjournal.io/blog/2022/08/12/workflows-not-starting

**Comment by mivano** [comment]
Nice overview. Can you also use the GH CLI to trigger a workflow? Saves auth header as the tool handles that.

**Reply by rajbos** [reply]
Good one! You can indeed, see the docs [here](https://cli.github.com/manual/gh_workflow_run).

I'll add that to the post as well, thanks!

**Reply by mivano** [reply]
Cool, using it more and more. Even from Github actions (do not forget to pass the token as an ENV). And you can always use the raw `gh api` command for stuff not (yet) implemented.

**Reply by rajbos** [reply]
Updated the post with this info. Thanks for the feedback!

**Comment by jessehouwing** [comment]
For the schedules... If the repo hasn't been updated recently, the schedule will stop. If the schedule has failed too often, the schedule will stop.

**Reply by mivano** [reply]
Do you know what is __recently__? Days, weeks? I cant find this easily. And does this not only applies to public repos?

I did see this as well: 

> Note: The schedule event can be delayed during periods of high loads of GitHub Actions workflow runs. High load times include the start of every hour. To decrease the chance of delay, schedule your workflow to run at a different time of the hour.

**Reply by jessehouwing** [reply]
I think it's 60 days.

**Reply by jessehouwing** [reply]
> Warning: To prevent unnecessary workflow runs, scheduled workflows may be disabled automatically. When a public repository is forked, scheduled workflows are disabled by default. In a public repository, scheduled workflows are automatically disabled when no repository activity has occurred in 60 days.

**Reply by rajbos** [reply]
Do note: there will be a notification by mail being sent to the user that last touched the schedule, to indicate it will be disabled soon. Will add that info as well 👍 .

**Reply by rajbos** [reply]
Added it to the post, thanks for letting me know!

**Comment by RubenGamarrarodriguez-tomtom** [comment]
Nice post.
The link to 
> When you follow **best practices** [...]

doesn't seem to go, it goes to https://devopsjournal.io/blog/2022/08/12/blog/2019/07/10/DevOps-Principles-series

it should go here?
https://devopsjournal.io/blog/2019/07/10/DevOps-Principles-series

maybe a missing forward slash `/` at the beginning so that the reference is absolute

----
Fixed it here https://github.com/rajbos/rajbos.github.io/pull/15

**Reply by rajbos** [reply]
Thanks a lot for the fix @RUBenGAMArrarodRiguEZ-ToMtOm ! Merged it in already 💪 .

**Comment by arihirsch** [comment]
This just saved me from a day of head banging.Thank you rob!

**Comment by maksimov** [comment]
Lovely overview! Thanks for writing it up.
